pkg/controller/controller_binding.go

```diff
@@ -365,13 +365,13 @@ func (c *controller) injectBinding(binding *v1alpha1.Binding, credentials map[st
 	}
 
 	found := false
-
 	_, err := c.kubeClient.Core().Secrets(binding.Namespace).Get(binding.Spec.SecretName, metav1.GetOptions{})
 	if err == nil {
 		found = true
 	}
 
 	if found {
+		// TODO: Check ownerReferences first?
```

Comment:
@ash2k also, please leave your suggestions in issue https://github.com/kubernetes-incubator/service-catalog/issues/978